agent/app/agent_capability.go

```diff
@@ -111,6 +114,10 @@ func (agent *ecsAgent) capabilities() ([]*ecs.Attribute, error) {
 	}
 
 	capabilities = agent.appendTaskENICapabilities(capabilities)
+	if agent.cfg.ENITrunkingEnabled {
```

Comment:
nit: i'd prefer to put this into appendENITrunkingCapabilities() like other method

Reply:
updated